fmriprep/interfaces/reports.py

```diff
@@ -218,10 +220,20 @@ def _generate_segment(self):
         else:
             dummy_scan_msg = dummy_scan_tmp.format(n_dum=self.inputs.algo_dummy_scans)
 
+        multiecho = "Single-echo EPI sequence."
+        n_echos = len(self.inputs.echo_idx)
+        if n_echos == 1:
+            multiecho = (
+                f"Multi-echo EPI sequence: only echo {self.inputs.echo_idx[0]} processed "
+                "in single-echo mode."
+            )
+        if n_echos > 2:
+            multiecho = (f"Multi-echo EPI sequence: {n_echos} echoes.")
```

Comment:
What about `n_echos == 2`? Perhaps that should be labeled `"Dual-echo EPI sequence"`.

Reply:
That will require a bit more of logic: at this point we should never see ``length == 2`` because dual echoes are split very early before this

Reply:
Ah, I see. Would it be better to not be explicit in the default then? I.e., not "single-echo EPI sequence", since it's not necessarily going to be single-echo? On the other hand, dual-echo is going to be pretty rare...

Reply:
In reality, this is to report how fMRIPrep worked - so I don't think it is inaccurate to say that the dataset was single-echo. Let's go with this and then refine is some user requests to have the possibility to see in the reports dual-echo as such.